emcmake.py

```diff
@@ -27,6 +27,9 @@ def run():
 
   args = sys.argv[1:]
 
+  # Restore removed SYSCONFIG settings so they will be present in the CMake environment
+  os.environ["_PYTHON_SYSCONFIGDATA_NAME"] = os.environ["_EMCMAKE_PYTHON_SYSCONFIGDATA_NAME"]
```

Comment:
Where exactly is `_PYTHON_SYSCONFIGDATA_NAME` removed?  Its removed by the `-E` we pass to python in the wrapper scripts?

Reply:
https://github.com/emscripten-core/emscripten/blob/d1530e7dd9c1d309fddd51b7d4b7201ec2cf3fd6/emcmake.bat#L13-L15

https://github.com/emscripten-core/emscripten/blob/d1530e7dd9c1d309fddd51b7d4b7201ec2cf3fd6/emcmake#L14-L16

Reply:
Ah interesting so this is a fight between two python-specific use cases. I vote to remove the stripping on `_PYTHON_SYSCONFIGDATA_NAME`.